create.html

```diff
@@ -29,6 +29,7 @@ <h3 class="Que"></h3>
 
       <p class="tags">Lorem, ipsum, dolor, sit,</p>
       <br />
+      <button class="main-card__button submit logout">Logout</button>
```

Comment:
Better call it `card` instead of `main-card`. It does not matter that it is on a `main` element (which it is, right?).